spec/factories/collection_factory_data.rb

```diff
@@ -594,6 +594,7 @@ def collection_one
     'Quality'            => 'Metadata quality summary',
     'UseConstraints' => {
       'Description' => 'These are some use constraints',
+      'FreeAndOpenData' => true,
```

Comment:
Is this correct?   Shouldn't this have a description and/or type?

Reply:
The type is not an actual field, the radio buttons are just so the form knows what fields to show or hide.
The description is just above.